std_msgs/Header header
int32 theta1
int32 theta2
int32 theta3